test/release/examples/users-guide/base/configs.execopts

```diff
@@ -0,0 +1,4 @@
+./a.out --n 1000
+./a.out --n=1000
+./a.out --epsilon=0.000001 --debug=true  # configs-eps+debug.good
+./a.out --epsilon=0.000001 --debug       # configs-eps+debug.good
```

Comment:
I cheated a little in the file by having main() take additional arguments and asserting that the argument was ./a.out... :)

But then I had to create the -fake version of the file anyway because of the need for the '# overrides for lines 3 and 4... :'(

An open question is how I make these -fake files maintainable over time...  sigh...